### ISSUE TITLE
tour: [MODIFICAR AMB UNA BREU DESCRIPCIÓ] Falta traducció

### ISSUE THREAD

**Issue by martisalcedo7** [zorion/go-tour-ca#3]
Context: https://go-tour-ca.appspot.com/flowcontrol/1

Una part del text no està traduïda al català. Apareix barrejada amb l'anglès.

**Comment by zorion** [comment]
Sí, quan van fer una restructuració del tour en anglès abans de fer la versió de go incompatible vaig pensar que valia la pena com a mínim tenir-ho en anglès amb les novetats, però no tinc temps de fer les traduccions. Aniré fent quan pugui. En tot cas, si algú em pot fer un pull request miraré d'acceptar-lo quan el vegi (que tampoc prometo que sigui inmediat). Moltes gràcies!!

**Comment by zorion** [comment]
Hem fet la traducció a una branca i l'estem desplegant, de moment aquí: https://20250307t185339-dot-go-tour-ca.appspot.com

Si es veu bé ho posarem a l'adreça principal.

**Comment by zorion** [comment]
Es veu bé però hem detectat un petit error i pujarem https://20250307t185907-dot-go-tour-ca.appspot.com abans de passar-ho a la principal.
